tools/ResourceMetaSchema.json

```diff
@@ -195,8 +195,7 @@
             }
           },
           "required": [
-            "oneOf",
```

Comment:
I wrote this ResourceMetaSchema.json file, so I think it's okay if I change it here. The description isn't really necessary at this point.